fang_examples/simple_async_worker/src/lib.rs

```diff
@@ -26,8 +26,10 @@ impl AsyncRunnable for MyTask {
         tokio::time::sleep(Duration::from_secs(3)).await;
 
         let new_task = MyTask::new(self.number + 1);
-        let metadata = serde_json::to_value(&new_task as &dyn AsyncRunnable).unwrap();
-        queue.insert_task(metadata, "common").await.unwrap();
+        queue
```

Comment:
please rebase the pr. it seems changes related to `insert_task` were already merged

Reply:
rebased